crates/flox/src/commands/search.rs

```diff
@@ -197,6 +202,16 @@ fn render_search_results_user_facing(
         writeln!(&mut writer, "{package:<column_width$}  {desc}")?;
     }
     writer.flush().context("couldn't flush search results")?;
+    if let Some(count) = search_results.count {
+        // Don't show the message if we have exactly the number of results as the limit,
+        // otherwise we would get messages like `Showing 10 of 10...`
+        if count != n_results as u64 {
+            eprint!(
+            "\nShowing {} of {} results. Use `flox search {{query}} --all` to see the full list.",
+            *SEARCH_LIMIT, count
+        );
```

Comment:
```suggestion
            eprint!(
            "\nShowing {} of {} results. Use `flox search {} --all` to see the full list.",
            *SEARCH_LIMIT, count, shell_escape::escape(search_term)
        );
```

Was there a particular reason _not_ to include the original query?
At least you could copy/paste this output

Reply:
I wrote it this way to be consistent with the `flox show {package}` message, but it looks like there was a PR that just merged that changed that as well so I'll fix this.